core/src/main/scala/org/apache/spark/internal/io/HadoopMapReduceCommitProtocol.scala

```diff
@@ -222,6 +222,7 @@ class HadoopMapReduceCommitProtocol(
 
   override def commitTask(taskContext: TaskAttemptContext): TaskCommitMessage = {
     val attemptId = taskContext.getTaskAttemptID
+    logDebug("Commit task")
```

Comment:
This seems more noisy than useful. Either remove or at least add something to the message that lets you identify what's being committed.

But in general I'd refrain from adding this kind of logging at the task level.

Reply:
ok, now I'm confused. I'd put this in the subclass, and was told "pull it up". Which I did. Now I'm told "remove". Fine.  Done. But it would be good for both of us to have some consistency here: me for less wasted effort, you for less reviewing.

Reply:
I've always complained about debug messages at task level. I'm pretty consistent about complaining about that because it makes debug logs less useful - it's ok with a few tasks but not with thousands.

Reply:
cut entirely